include/p4mlir/Dialect/P4HIR/P4HIR_Ops.td

```diff
@@ -1694,4 +1694,79 @@ def ArraySetOp : P4HIR_Op<"array_set",
 include "p4mlir/Dialect/P4HIR/P4HIR_ParserOps.td"
 include "p4mlir/Dialect/P4HIR/P4HIR_ControlOps.td"
 
+def BrOp : P4HIR_Op<"br",
+    [DeclareOpInterfaceMethods<BranchOpInterface, ["getSuccessorForOperands"]>,
+     Pure, Terminator]> {
+  let summary = "Unconditional branch";
+  let description = [{
+    The `p4hir.br` branches unconditionally to a block.
+
+    Example:
+
+    ```mlir
+      ...
+        p4hir.br ^bb3
+      ^bb3:
+        p4hir.return
+    ```
+  }];
+
+  let builders = [
+    OpBuilder<(ins "mlir::Block *":$dest,
+              CArg<"mlir::ValueRange", "{}">:$destOperands), [{
+      $_state.addSuccessors(dest);
+      $_state.addOperands(destOperands);
+    }]>
+  ];
+
+  let arguments = (ins Variadic<AnyP4Type>:$destOperands);
+  let successors = (successor AnySuccessor:$dest);
+  let assemblyFormat = [{
+    $dest (`(` $destOperands^ `:` type($destOperands) `)`)? attr-dict
+  }];
+}
+
+def BrCondOp : P4HIR_Op<"brcond",
```

Comment:
Since there is a very similar [`cf.br_cond`](https://mlir.llvm.org/docs/Dialects/ControlFlowDialect/#cfcond_br-cfcondbranchop), is it better to name this `p4hir.br_cond`?